website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,764 @@
+				[release highlight](/highlights/2022-03-31-native-event-codecs) for more about this new feature and how
+				to use it.
+				"""
+			pr_numbers: [12048]
+		},
+		{
+			type: "enhancement"
+			scopes: ["datadog_metrics sink"]
+			description: """
+				The `datadog_metrics` sink now supports sending aggregated summary metrics (typically scraped from
+				a Prometheus exporter) to Datadog. Previously these metrics were dropped at the sink.
+				"""
+			pr_numbers: [12436]
+		},
+		{
+			type: "enhancement"
+			scopes: ["datadog_metrics sink"]
+			description: """
+				The `datadog_metrics` sink now supports sending aggregated summary metrics (typically scraped from
+				a Prometheus exporter) to Datadog. Previously these metrics were dropped at the sink.
+				"""
+			pr_numbers: [12436]
+		},
+		{
+			type: "fix"
+			scopes: ["internal_logs source"]
+			description: """
+				The `internal_logs` source occasionally missed some events generated early in Vector's start-up, before
+				the component was initialized. This was remedied so that the `internal_logs` source more reliably
+				captures start-up events.
+				"""
+			pr_numbers: [12411]
+		},
+		{
+			type: "feat"
+			scopes: ["gcp_pubsub source", "sources"]
+			description: """
+				A new `gcp_pubsub` source was added for consuming events from [GCP PubSub](https://cloud.google.com/pubsub).
+				"""
+			pr_numbers: [12057]
+		},
+		{
+			type: "feat"
+			scopes: ["websocket sink", "sinks"]
+			description: """
+				A new `websocket` sink was added for sending events to a remote websocket listener.
+				"""
+			pr_numbers: [9632]
+			contributors: ["@zshell31"]
+		},
+		{
+			type: "fix"
+			scopes: ["vrl"]
+			description: """
+				The `parse_ruby_hash` VRL function can now parse hashes that contain a symbol as the value, such as `{ "key" => :foo }`.
+				"""
+			pr_numbers: [12514]
+		},
+		{
+			type: "enhancement"
+			scopes: ["releasing"]
+			description: """
+				The RPM package now adds the created `vector` user to the `systemd-journal-remote` group to be able to
+				consume journald events from a remote system. This matches the Debian package.
+				"""
+			pr_numbers: [12563]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now allows configuration of `extra_namespace_label_selector` which Vector
+				will to use select the pods to capture the logs of, if set, based on labels attached to the pod
+				namespace. This is similar to the `extra_label_selector` option which applies to pod labels.
+				"""
+			contributors: ["@anapsix"]
+			pr_numbers: [12438]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now reads events in order whenever a pod log file rotates. Previously Vector could start reading the new file before it finished processing the previous one, resulting in the logs being out-of-order.
```

Comment:
Not sure what the formatting rules are for Cue docs, but this line, and the one on 354 violates line-length rules (assuming one exists, if not, it really should exist 😄)